### PR TITLE
Add :flag command

### PR DESCRIPTION
**Pull request by teamshortcut**
Add a :flag command, which functions in the same way as :level but will return a random base64 string "flag"

### CODE REVIEW

**Comment by thebeanogamer** [comment]
[Do We Need This?](http://www.youtube.com/watch?v=MRyaOZQn1FM "Do We Need This?")
Muse - Do We Need This?

**Comment by teamshortcut** [comment]
"Need" is a strong word.

**Comment by thebeanogamer** [comment]
image

**Comment by thebeanogamer** [comment]
image

**Comment by teamshortcut** [comment]
Let me paint you a picture:
GenericDiscordMember - plz hints on 4.3 I've tried literally everything will exchange hints or answers I've done everything up to Level 2
OtherLessGenericDiscordMember - :flag hq 4 3

Problem solved.

**Comment by thebeanogamer** [comment]
Right...
image

**Comment by teamshortcut** [comment]
Perfect example usecase:
image

**Comment by thebeanogamer** [comment]
image

**Comment by teamshortcut** [comment]
Hey the code is high quality, it's essentially the same as the :level command so most of it is what @picapi wrote.

**Comment by Bentechy66** [comment]
I mean the code is high quality (although it is somewhat duplicated) and it's an epic meme:tm: I can somewhat see a niche use case here.

**Comment by thebeanogamer** [comment]
I think we would all like to know @shadycake's position on this, and it would be irresponsible to merge it without his approval.

File: bot/cogs/cyber.py [hunk 1]
```diff
@@ -121,6 +134,68 @@ def __init__(self, bot: Bot):
...
+        Gets information about a specific CyberStart Game level and challenge.
+        If the date is before the start date of game (15th January 2019) it will redirect to game() instead
+        """
+
+        if datetime.date.today() < datetime.date(2019, 1, 15):
+            await self.game.callback(self, ctx)
+            return
+
+        # Gather data from CyberStart Game.
+        with open("bot/data/game.json") as f:
```
Comment (AnonGuy) [inline bot/cogs/cyber.py]: I really don't think we need to load the titles/descriptions or check that the levels are correct for this meme command.
Could you rewrite this so it only includes the relevant code? (lines 181 - 197)

Comment (thebeanogamer) [reply bot/cogs/cyber.py]: I agree with this, and as such shall change my review to concur.

File: bot/cogs/cyber.py [hunk 2]
```diff
@@ -12,6 +12,19 @@
...
 
 
+async def generatebase64(seed: int):
+    import random
+    import string
+    random.seed(seed)
+    letters = string.ascii_letters + string.digits + "+" + "/" + "="
+    result = ""
+
+    for _ in range(20):
```
Comment (AnonGuy) [inline bot/cogs/cyber.py]: You can use the `k` parameter of `random.choices` without needing to use a for loop:
image

**Pull request merged**